examples/machinelearningservices/synapsespark.yaml

```
@@ -0,0 +1,198 @@
+  forProvider:
+    applicationInsightsIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example
+    identity:
+    - type: SystemAssigned
+    keyVaultIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: synapsesparkvault
+    location: West Europe
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgsynapsespark
+    storageAccountIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example0001
+---
+apiVersion: azure.upbound.io/v1beta1
+kind: ResourceGroup
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/synapsespark
+  labels:
+    testing.upbound.io/example-name: rgsynapsespark
+  name: rgsynapsespark
+spec:
+  forProvider:
+    location: West Europe
+---
+apiVersion: storage.azure.upbound.io/v1beta1
+kind: Account
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/synapsespark
+  labels:
+    testing.upbound.io/example-name: example0001
+  name: example0001
+spec:
+  forProvider:
+    accountReplicationType: LRS
+    accountTier: Standard
+    location: West Europe
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgsynapsespark
+---
+apiVersion: storage.azure.upbound.io/v1beta1
+kind: DataLakeGen2FileSystem
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/synapsespark
+  labels:
+    testing.upbound.io/example-name: datalakegenfs
+  name: datalakegenfs
+spec:
+  forProvider:
+    storageAccountIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: example0001
+---
+apiVersion: synapse.azure.upbound.io/v1beta1
+kind: SparkPool
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/synapsespark
+  labels:
+    testing.upbound.io/example-name: example
+  name: example
+spec:
+  forProvider:
+    nodeCount: 3
+    nodeSize: Small
+    nodeSizeFamily: MemoryOptimized
+    synapseWorkspaceIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: exampleworkspace001
+---
+apiVersion: synapse.azure.upbound.io/v1beta1
+kind: Workspace
+metadata:
+  annotations:
+    meta.upbound.io/example-id: machinelearningservices/v1beta1/synapsespark
+  labels:
+    testing.upbound.io/example-name: exampleworkspace001
+  name: exampleworkspace001
+spec:
+  forProvider:
+    identity:
+    - type: SystemAssigned
+    location: West Europe
+    resourceGroupNameSelector:
+      matchLabels:
+        testing.upbound.io/example-name: rgsynapsespark
+    sqlAdministratorLogin: sqladminuser
+    storageDataLakeGen2FilesystemIdSelector:
+      matchLabels:
+        testing.upbound.io/example-name: datalakegenfs
+---
+apiVersion: alertsmanagement.azure.upbound.io/v1beta1
+kind: MonitorSmartDetectorAlertRule
+metadata:
+  annotations:
+    meta.upbound.io/example-id: alertsmanagement/v1beta1/monitorsmartdetectoralertrule
```

```suggestion
    meta.upbound.io/example-id: machinelearningservices/v1beta1/synapsespark
```

Could you please fix the example-id?

---

Done